crowbar_framework/app/views/installer/upgrades/start.html.haml

```diff
@@ -0,0 +1,30 @@
+.row
+  .col-lg-12
+    = form_tag start_upgrade_path, method: "post" do
+      .panel.panel-default
+        .panel-heading
+          %h2
+            = t("installer.upgrades.header", product: branding_config.page_title)
+
+        .panel-plain.wizard
+          = render partial: "nav", locals: { active: :start }
+
+          .wizard-body.col-lg-9
+            %h2
+              = t(".header")
+
+            %p
+              = t(".description")
+
+        .panel-footer
+          .row
+            .col-lg-3
+              = progress_steps 3, 1, 8
+
+            .col-lg-9.text-right
+              .btn-group{ role: "group" }
+                = link_to icon_tag(:chevron_left, t(".cancel")),
+                  installer_root_path,
+                  class: "btn btn-default"
+                = button_tag icon_tag(:chevron_right, t(".continue")),
+                  class: "btn btn-primary"
```

Comment:
The continue button here should actually bring you to the UI for uploading the tar ball and restoring that (i.e. running the installer), currently it just leads to the repo check UI.